x-pack/plugins/lens/public/visualizations/datatable/components/table_basic.tsx

```diff
@@ -271,7 +272,7 @@ export const DatatableComponent = (props: DatatableRenderProps) => {
       firstLocalTable.columns.reduce<Record<string, boolean>>(
         (map, column) => ({
           ...map,
-          [column.id]: column.meta.type === 'number',
+          [column.id]: column.meta.type === 'number' && column.meta.params?.id !== 'range',
```

Comment:
do you think it is worth to integrate this kind of logic into a shared util?

Reply:
Moved to shared util in 0ab6417